src/content-script.ts

```diff
@@ -27,8 +29,18 @@ import { isBodyWithReplies, isRepositoryDetails } from "./validations";
 	}
 	const { default_branch: defaultBranch } = repositoryDetails;
 
+	// 2.5 get the issue JSON
+	// https://api.github.com/repos/JoshuaKGoldberg/refined-saved-replies/issues/45
+	const issueOrPRDetails = (await (
+		await fetch(
+			`https://api.github.com/repos/${userOrOrganization}/${repository}/issues/${issueOrPR}`
+		)
+	).json()) as unknown;
+
 	// 3. Fetch the repository's .github/replies.yml
-	const repliesUrl = `https://raw.githubusercontent.com/${userOrOrganization}/${repository}/${defaultBranch}/.github/replies.yml`;
+	// const repliesUrl = `https://raw.githubusercontent.com/${userOrOrganization}/${repository}/${defaultBranch}/.github/replies.yml`;
+	const repliesUrl = `https://gist.githubusercontent.com/Pinjasaur/7364ffd89e01047ace51a5ee368d9a4d/raw/fd9fe1dee19bd0fee682fc4b1dfea25ebe42ad99/replies.yml`;
```

Comment:
note this is hardcoded to a test `replies.yml`, should be reverted before merging